code/src/index.css

```diff
@@ -5,9 +5,77 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: black;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+h1 {
+ font-size: 24px;
+ color: #fff;
+}
+
+h2 {
+  font-size: 14px;
+  color: #fff;
+  font-weight: 200;
+}
+
+p {
+  font-size: 16px;
+  font-weight: 200;
+}
+
+
+.outerDiv {
+ display: flex;
+ flex-wrap: wrap;
+ margin: 0;
+ width: 100%;
+}
+
+.outerDiv :hover {
+  opacity: 0.5;
+}
+
+.movieDisplay {
+  position: relative;
```

Comment:
remember to add some kind of width here since they should always be 4 movies in each row.